presto-main/src/main/java/com/facebook/presto/operator/scalar/DateTimeFunctions.java

```diff
@@ -167,7 +169,7 @@ public static double toUnixTimeFromTimestampWithTimeZone(@SqlType(StandardTypes.
     }
 
     @ScalarFunction("to_iso8601")
-    @SqlType(StandardTypes.VARCHAR)
+    @SqlType(VarcharType.VARCHAR_MAX_LENGTH)
```

Comment:
I think we can determine max characters count for date or timestamp in `iso8601` format. Can't we?